static/hello-world/src/App.tsx

```diff
@@ -1,18 +1,18 @@
 // @ts-nocheck
 import React, { useEffect, useState, useRef, useCallback } from "react";
-// import { invoke } from "@forge/bridge";
+import { invoke } from "@forge/bridge";
 import Canvas from "./Canvas";
 
 function App() {
   const [data, setData] = useState(null);
 
-  //   useEffect(() => {
-  //     invoke("getText", { example: "my-invoke-variable" }).then(setData);
-  //   }, []);
+  useEffect(() => {
```

Comment:
any changes here must be built using `npm run build` to be picked up by forge app, while running `forge tunnel` in root folder.